src/main/java/lesson3/task4/Conversion.java

```diff
@@ -0,0 +1,10 @@
+package lesson3.task4;
+
+public class Conversion {
```

Comment:
In your potential system you can have many convertions: Currency convertion, Units convertions etc. You should name your classes in such a way that we will know exactly with what we are delaing with. Here in my opinion it should be sth like: TimeConverter